solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -611,8 +621,17 @@ contract RandomBeacon is Ownable {
         }
 
         if (relay.requestCount % groupCreationFrequency == 0) {
-            // TODO: Once implemented, invoke:
-            // createGroup(uint256(keccak256(entry)));
+            // Just in case, make sure the group creation can be triggered to
+            // avoid unexpected revert. There is a possibility entries are
+            // submitted very quickly and if the group creation frequency
+            // is small enough, group creations may overlap.
+            if (dkg.currentState() == DKG.State.IDLE) {
+                // slither-disable-next-line reentrancy-events
+                createGroup(uint256(keccak256(entry)));
+            } else {
+                // slither-disable-next-line reentrancy-events
+                emit DkgStartFailed();
```

Comment:
I would not consider it as any kind of a failure, it's a normal situation.

Reply:
Yeah, this event is just in case and can make debugging easier. Do you think we should get rid of it?

Reply:
Yeah, I'd remove it to not confuse anyone. It's also some additional cost - really small but why pay it given this situation is normal. 🤷🏻

Reply:
https://github.com/keep-network/keep-core/pull/2691/commits/eb221288d15eb715257eac977f68bb68efc20f5e